nodebuilder/blob/cmd/blob.go

```diff
@@ -41,15 +41,18 @@ func init() {
 		&fee,
 		"fee",
 		-1,
-		"specifies fee for blob submission",
+		"specifies fee (in TIA) for blob submission [optional]",
```

Comment:
Hmmmm. So one thing that's tricky about this is that an astute reader will notice the type is `int` instead of `uint`, and will naturally want to know what will happen if a negative value is set. IMO this should be documented.

Reply:
done